### ISSUE TITLE
[BUG] CreateSource request: Issue with request body

### ISSUE THREAD

**Issue by daclear** [BarRaider/obs-websocket-dotnet#64]
**Issue Type** 
- Bug 

**Describe the bug**
When I'm using the CreateSource request from [OBSWebsocket_Requests.cs] it turned out that the request gets exepted from OBS Websocket Plugin and the source gets created but the settings of the source never gets apply. 
As an example I used the "window_capture" - type and a Postman window as a source.
With Postman webSocket functionality it worked just fine with this command:
```
{
      "sourceName": "Postman",
      "sourceKind": "window_capture",
      "sceneName": "Capture_Screen",
      "sourceSettings": {
        "window": "Postman:Chrome_WidgetWin_1:Postman.exe"
      },
      "setVisible": true,
    "request-type": "CreateSource",
    "message-id": "1"
}
```
But in my .net Application it doesn't. (see image below) 

**To Reproduce**
1. Initialize a OBSWebsocket connection (required: OBS websocket plugin and running OBS Studio)
2. Send "CreateSource" request 
3. check the result in OBS Studio

**Expected behavior**
The request "CreateSource" should create the defined source in OBS Studio and the settings should be applied correctly (see image below)

**Screenshots**

***Issue***
![Issue](https://user-images.githubusercontent.com/14124077/125281596-b6c3c100-e316-11eb-943d-f5f9f2b98031.png)

***Expected Result***
![Expected Result](https://user-images.githubusercontent.com/14124077/125283485-c0e6bf00-e318-11eb-8f07-3e8ae9313941.png)

**Versions**
OBS Version: 27.0.1
OBS WebSocket Version: 4.9.1
OBS WebSocket Dotnet (this library) Version: 4.9.0
OS: Windows 10

**Additional context**
After a short investigation I figured out that there is a space in two places of the request body which are part of the "CreateSource" request. 

https://github.com/BarRaider/obs-websocket-dotnet/blob/268b7f6c52d8daf8e8d08cf517812009c6f9cc26/obs-websocket-dotnet/OBSWebsocket_Requests.cs#L1951

https://github.com/BarRaider/obs-websocket-dotnet/blob/268b7f6c52d8daf8e8d08cf517812009c6f9cc26/obs-websocket-dotnet/OBSWebsocket_Requests.cs#L1956

This leeds to the result that it will be append a '\t' to the key string. 
I logged the object that I sent to the WebSocket and this confirmed my hypothesis

```
 {
      "sourceName": "Postman",
      "sourceKind": "window_capture",
      "sceneName": "Capture_Screen",
      "sourceSettings\t": {
        "window": "Postman:Chrome_WidgetWin_1:Postman.exe"
      },
      "setVisible\t": true
    }

```

**Comment by BarRaider** [comment]
Fixed in 7314bb on the dev-4.9.1 branch.
Will be merged to master in next release

**Comment by BarRaider** [comment]
Resolved in 4.9.1-Beta1 release
